torch/_inductor/ir.py

```diff
@@ -3718,13 +3716,15 @@ def __init__(
                     # repeat_interleave(const at::Tensor & repeats, c10::optional<int64_t> output_size=c10::nullopt)
                     # repeat_interleave(const at::Tensor & self, int64_t repeats,
                     #       c10::optional<int64_t> dim=c10::nullopt, c10::optional<int64_t> output_size=c10::nullopt)
-                    self.kernel = (
-                        f"at::_ops::{kernel.__name__.replace('.default', '')}::call"
-                        if kernel._overloadname == "default"
-                        else f"at::_ops::{kernel.__name__.replace('.', '_')}::call"
-                    )
+                    # self.kernel = (
+                    #     f"at::_ops::{kernel.__name__.replace('.default', '')}::call"
+                    #     if kernel._overloadname == "default"
+                    #     else f"at::_ops::{kernel.__name__.replace('.', '_')}::call"
+                    # )
+                    self.kernel = f"at::{op_base_name}"
```

Comment:
@desertfire put this here because we had trouble resolving overloads, is that not a problem anymore?

Reply:
Here's my current workaround: 

.default overload can directly call `at::{op_base_name}`
non .default overload has to call `at::_op::{op_base_name}_{overload_name}::call`. 

In my next PR, I will redirect .default overload  to also call `at::_op::{op_base_name}::call`, when we populate missing optional arguments.